internal/observability/graph.go

```diff
@@ -38,7 +38,9 @@ var (
 	_defaultGraphSize = 128
 	// Latency buckets for histograms. At some point, we may want to make these
 	// configurable.
-	_bucketsMs = bucket.NewRPCLatency()
+	_bucketsMs = append(bucket.NewRPCLatency(),
+		20000, 30000, 40000, 50000, 60000, 70000, 80000, 90000, 100000, 110000, 120000,
+	)
```

Comment:
Should we create a new methods for configurable buckets as the comment propose?

Reply:
Good idea, created a follow up PR: https://github.com/yarpc/yarpc-go/pull/2105